README.md

```diff
@@ -1 +1,32 @@
-# angular-hello-world-lab
\ No newline at end of file
+# Hello World 
+
+## Objectives
+
+- Create a Controller
+- Create an element for Controller binding
+- Create a public $scope method
+- Display the $scope value in the View
+
+## Instructions
+
+Setup the directory structure as follows:
```

Comment:
Once the students have built out the file structure, it's okay to give them some empty files and folders. This helps them focus on the learning objectives for that particular lab without having to build out an app from near-scratch every time. 

In this case, go ahead and include the empty app.js  and MainController.js files.